CONTRIBUTING.md

```diff
@@ -384,7 +384,7 @@ function iDoSomething(uint awesome_argument) {
 **Modifiers in long function declarations**
```

Comment:
Maybe just remove this part altogether, since we say above that we use the solidity style guide by default.

Reply:
nah it's good to be explicit

Reply:
Hmm yeah you are correct, not picky. I reread it again and it should just go. It's done.